src/main/fc/config.c

```diff
@@ -410,6 +410,11 @@ static void validateAndFixConfig(void)
 #ifndef USE_OSD_SLAVE
 void validateAndFixGyroConfig(void)
 {
+    // Disable dynamic filter if gyro loop is less than 2KHz
+    if (gyro.targetLooptime > GYRO_LOOPTIME_2KHZ) {
```

Comment:
We have new macro for such calculations so this should look better:
`HZ_TO_INTERVAL_US(2000)`

Reply:
I would extract this condition to a separate function and place it in `gyroanalyse.c` to be called in two places of this PR